test/acceptance/cli-test.js

```diff
@@ -13,10 +13,12 @@ describe('ember-template-lint executable', function () {
   beforeEach(function () {
     project = Project.defaultSetup();
     project.chdir();
+    process.env.LC_ALL = 'en_US';
```

Comment:
Can you move this up to just below the `describe` and do:

```js
setupEnvVar('LC_ALL', 'en_US');
```

This will ensure that it is properly reset after each test (to either be missing, or the original value).

Reply:
Didn't see that, nice 👍